_posts/2020-07-06-concurrency-costs.md

```diff
@@ -588,7 +588,7 @@ Here's a quick look at some usual and unusual ways of achieving levels lower on
 
 #### Level 4
 
-You probably don't want to really be in level 4 but it's certainly better than level 5. So, if you still have your job and your users haven't all abandoned you, it's usually pretty easy to get out of level 5. More than half the battle is just recognizing what's going on and from there the solution is often clear. Many times, you've simply violated some rule like "don't use pure spinlocks in userspace" or "you built a spinlock by accident" or "so-and-so accidentally held that core lock during IO". There's never almost never any inherent reason you'd need to stay in level 5 and you can usually find an almost tradeoff-free fix.
+You probably don't want to really be in level 4 but it's certainly better than level 5. So, if you still have your job and your users haven't all abandoned you, it's usually pretty easy to get out of level 5. More than half the battle is just recognizing what's going on and from there the solution is often clear. Many times, you've simply violated some rule like "don't use pure spinlocks in userspace" or "you built a spinlock by accident" or "so-and-so accidentally held that core lock during IO". There's almost never any inherent reason you'd need to stay in level 5 and you can usually find an almost tradeoff-free fix.
```

Comment:
for whatever reason GitHub doesn't propery highlght the removal of duplicate words? Very annoying:

![image](https://user-images.githubusercontent.com/2403521/86661034-4c689a00-bfb9-11ea-9dc8-25abd880df47.png)

Normally the removed part is highlighted but not in these cases for some reason.

Reply:
Yeah I noticed that too, pretty irritating. Maybe it's the line length?

Reply:
> Maybe it's the line length?

Yeah could be. I couldn't find a pattern. I just gave up when it didn't show and trusted you at some point. `git diff` after pulling down the changes works fine though.